app/models/concerns/searchable.rb

```diff
@@ -0,0 +1,67 @@
+module Searchable
+  extend ActiveSupport::Concern
```

Comment:
Yeah, I think something that involved both would wind up in Searchable, but I'm admittedly not sure how to resolve ties here. Thoughts?